python/hail/methods/statgen.py

```diff
@@ -1596,6 +1597,8 @@ def split_multi_hts(ds, keep_star=False, left_aligned=False) -> MatrixTable:
         If ``True``, variants are assumed to be left
         aligned and have unique loci. This avoids a shuffle. If the assumption
         is violated, an error is generated.
+    vep_root : :obj:`str`
```

Comment:
can you put a sentence in the docs about what this does?

Reply:
Is this the way you want this? I was toying with adding an expression, but that would be different from all the others (GQ, etc). I wonder if it's even worth an argument

Reply:
passing in an expression is no easier than doing an `annotate_rows`, which would be equivalent. I think this is nice and simple